cmd/loop/loopout.go

```diff
@@ -235,6 +243,15 @@ func loopOut(ctx *cli.Context) error {
 		}
 	}
 
+	paymentTimeout := 0
+	if ctx.IsSet("payment_timeout") {
+		paymentTimeout = int(ctx.Duration("payment_timeout").Seconds())
+		if paymentTimeout < 0 {
```

Comment:
If user passes `--payment_timeout=0s`, it should also fail, I think. So this check should be `if paymentTimeout <= 0`